apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/RetryTimings.cs

```diff
@@ -0,0 +1,124 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using System;
+
+namespace Google.Cloud.Storage.V1;
+
+/// <summary>
+/// These specify the custom timing configurations for retrying in case of failure of the API request.
+/// </summary>
+public sealed class RetryTimings
+{
+    /// <summary>
+    /// The backoff time between the first attempt and the first retry. It must be non-negative. 
+    /// </summary>
+    public TimeSpan InitialBackoff { get; private set; }
+
+    /// <summary>
+    /// Maximum backoff time between retries. It must be at least as much as initialBackoff.
+    /// </summary>
+    public TimeSpan MaxBackoff { get; private set; }
+
+    /// <summary>
+    /// The multiplier to apply to the backoff on each iteration. It is always greater than or equal to 1.0.
+    /// </summary>
+    public double BackoffMultiplier { get; private set; }
+
+    /// <summary>
+    /// The default initial backoff time between the first attempt and the first retry. It is set to 1 second by default.
+    /// </summary>
+    public static TimeSpan DefaultInitialBackoff { get; } = TimeSpan.FromSeconds(1);
+
+    /// <summary>
+    /// The default maximum backoff time between retries. It is set to 32 seconds by default.
+    /// </summary>
+    public static TimeSpan DefaultMaxBackoff { get; } = TimeSpan.FromSeconds(32);
+
+    /// <summary>
+    /// The default maximum backoff multiplier to be applied on each iteration. It is set to 2 by default.
+    /// </summary>
+    public static double DefaultBackoffMultiplier { get; } = 2;
+
+    /// <summary>
+    /// Returns the retry timing configurations all set to default.
+    /// </summary>
+    public static RetryTimings Default { get; } = new();
+
+    internal RetryTimings Clone() => new()
+    {
+        InitialBackoff = InitialBackoff,
+        MaxBackoff = MaxBackoff,
+        BackoffMultiplier = BackoffMultiplier
+    };
+
+    /// <summary>
+    /// Returns the existing retry timings configurations using the specified initial backoff.
+    /// </summary>
+    /// <param name="initialBackoff">Custom initial Backoff for retry.</param>
+    /// <returns>Retry timings with updated initial backoff.</returns>
```

Comment:
```suggestion
    /// <returns>A new instance based on the existing values, but with the specified initial backoff.</returns>
```

"Update" sounds like the existing instance will be modified.